presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ConfigAPI.java

```diff
@@ -41,12 +41,12 @@
 @Singleton
 public final class ConfigAPI
 {
-    private final APIFileHandler apiFileHandler;
+    private final ApiFileHandler apiFileHandler;
```

Comment:
If you are changing `API` -> `Api` change it in all the files ConfigAPI, PackageAPI, ..